Description: G_INLINE_FUNC macro is deprecated, remove it
 GCC-9 deprecated detection is much harder, and even error disables won't
 catch deprecated macros. Per [1] this can now be safely replaced
 with "static inline"
 [1]: https://developer.gnome.org/glib/stable/glib-Miscellaneous-Macros.html#G-INLINE-FUNC:CAPS
Forwarded: yes (https://github.com/vmware/open-vm-tools/issues/367)
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1842301
Last-Update: 2019-09-03

--- a/open-vm-tools/lib/include/vmware/tools/plugin.h
+++ b/open-vm-tools/lib/include/vmware/tools/plugin.h
@@ -281,7 +281,7 @@ typedef struct ToolsAppCtx {
  *
  * @return TRUE if COM is initialized when the function returns.
  */
-G_INLINE_FUNC gboolean
+static inline gboolean
 ToolsCore_InitializeCOM(ToolsAppCtx *ctx)
 {
    if (!ctx->comInitialized) {
--- a/open-vm-tools/lib/include/vmware/tools/threadPool.h
+++ b/open-vm-tools/lib/include/vmware/tools/threadPool.h
@@ -91,7 +91,7 @@ typedef struct ToolsCorePool {
  *******************************************************************************
  */
 
-G_INLINE_FUNC ToolsCorePool *
+static inline ToolsCorePool *
 ToolsCorePool_GetPool(ToolsAppCtx *ctx)
 {
    ToolsCorePool *pool = NULL;
@@ -123,7 +123,7 @@ ToolsCorePool_GetPool(ToolsAppCtx *ctx)
  *******************************************************************************
  */
 
-G_INLINE_FUNC guint
+static inline guint
 ToolsCorePool_SubmitTask(ToolsAppCtx *ctx,
                          ToolsCorePoolCb cb,
                          gpointer data,
@@ -153,7 +153,7 @@ ToolsCorePool_SubmitTask(ToolsAppCtx *ct
  *******************************************************************************
  */
 
-G_INLINE_FUNC void
+static inline void
 ToolsCorePool_CancelTask(ToolsAppCtx *ctx,
                          guint taskId)
 {
@@ -197,7 +197,7 @@ ToolsCorePool_CancelTask(ToolsAppCtx *ct
  *******************************************************************************
  */
 
-G_INLINE_FUNC gboolean
+static inline gboolean
 ToolsCorePool_StartThread(ToolsAppCtx *ctx,
                           const gchar *threadName,
                           ToolsCorePoolCb cb,
